src/mongocxx/options/create_collection.cpp

```diff
@@ -133,12 +134,22 @@ bsoncxx::document::value create_collection::to_document() const {
     }
 
     if (_validation) {
+        BSONCXX_SUPPRESS_DEPRECATION_WARNINGS_BEGIN;
```

Comment:
Do you still need to do this if you updated the validation class?

Reply:
Right, and here is where you could call the `no_deprecated` or `no_deprecated_warning` version of the function.